src/submission/container.jsx

```diff
@@ -47,15 +47,23 @@ const renderByCode = (code, page, message) => {
     } else if (
       ['syntacticalvalidity', 'quality', 'macro'].indexOf(page) !== -1
     ) {
-      if (code > VALIDATING) {
-        toRender.push(<Edits />)
-      }
+      toRender.push(<Edits />)
     } else if (page === 'submission') {
-      if (code > VALIDATING) {
-        toRender.push(<IRSReport />)
-        toRender.push(<Summary />)
-        toRender.push(<Signature />)
+      let warningOrReceipt = null
+
+      if (code !== SIGNED) {
+        warningOrReceipt = <ReadyToSign />
```

Comment:
I think I prefer having the ReceiptContainer below the Signature component so it matched how the quality and macro verifiers work (and in general I think it's more logical for a new thing to be inserted below/at the bottom of the page). I missed the receipt and thought the request was hanging on my first submission.

Reply:
Yep, just wasn't thinking about that part. Will fix that.